samples/dialogs/dialogs.h

```diff
@@ -28,6 +28,12 @@ of MSW, MAC and OS2
     #define USE_WXUNIVERSAL 0
 #endif
 
+#ifdef WXUSINGDLL
```

Comment:
This breaks the CMake build again because the generic dialogs are not compiled. The correct fix is to add the sources of the generic dialogs to the ` dialogs` sample. I'll make a PR for this.

Reply:
While fixing this I noticed `"TOOLKIT=='PM'` in `dialogs.bkl`. It doesn't appear in any other file. Out of interest, what is this? And should I remove it?

Reply:
Yes, please remove it, it's a trace of unlamented OS/2 support (its shell was called Presentation Manager, hence PM).